src/Components/WebView/WebView/test/Program.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Http;
+using Microsoft.AspNetCore.Components.WebView.Photino;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Components.WebView;
+
+class Program
+{
+    // Yes, this is a Program.Main() inside of a test project! This project is a regular xUnit.net test project, but
+    // some of the WebViewManagerTests also launch this project as a regular executable to launch UI tests. To achieve
+    // this, the CSPROJ has the <StartupObject> property set to indicate that _this_ is the Program.Main() to use
+    // when launching as an executable.
```

Comment:
Like above, further reasons for this E2E stuff to be separated from the regular unit tests project.